bin/liquidator/README.md

```diff
@@ -1,3 +1,29 @@
+## Disclaimer
+
+The following open source code contains an automated bot designed to perform liquidations. Please note that the use of this code is at your own risk and responsibility.
```

Comment:
would rather call it an example to document how to use the API

Reply:
done